zone/zoning.cpp

```diff
@@ -961,6 +961,16 @@ void Client::GoToBind(uint8 bindnum) {
 		MovePCGuildID(m_pp.binds[bindnum].zoneId, GUILD_NONE, m_pp.binds[bindnum].x, m_pp.binds[bindnum].y, m_pp.binds[bindnum].z, m_pp.binds[bindnum].heading, 1);
 }
 
+void Client::BootFromGuildInstance() {
+	ZoneBanishPoint* zbp = zone->GetZoneBanishPoint();
+	if (zbp == nullptr) {
+		this->GoToBind();
+	}
+	else {
+		MovePCGuildID(zbp->target_zone_id, GUILD_NONE, zbp->x, zbp->y, zbp->z, zbp->heading, 1, ZoneSolicited);
```

Comment:
ZoneSolicited does not need to be defined here. Mirror above 'gotobind' function.